### PR TITLE
Sensu plugin to check if MySQL hosts from performance_schema.host_cache are being blocked

### PR DESCRIPTION
**Pull request by lardcanoe**

### CODE REVIEW

**Comment by lardcanoe** [comment]
Build failure is unrelated

File: plugins/mysql/mysql-host-errors.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env ruby
+#
+#   <script name>
+#
+# DESCRIPTION:
+#   what is this thing supposed to do, monitor?  How do alerts or
```
Comment (tas50) [inline plugins/mysql/mysql-host-errors.rb]: Can you fill th header out here so it's clear what the plugin does and how it should be used?

File: plugins/mysql/mysql-host-errors.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+#   gem: <?>
+#
+# USAGE:
+#   example commands
+#
+# NOTES:
+#   Does it behave differently on specific platforms, specific use cases, etc
+#
+# LICENSE:
+#   <your name>  <your email>
```
Comment (tas50) [inline plugins/mysql/mysql-host-errors.rb]: Make sure to put your contact info here

File: plugins/mysql/mysql-host-errors.rb [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+# ===
+#
+# This plugin counts the number of hosts violating a certain error
+#
+# Copyright 2015 Steve Frank <lardcanoe@gmail.com>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (tas50) [inline plugins/mysql/mysql-host-errors.rb]: The ruby gems require is no longer necessary. You can remove this.

**Comment by lardcanoe** [comment]
@tas50 Updated based on your comments. Please add to the contrib file that there are plenty of scripts that are violating the process so don't blindly copy/paste them... for instance I used mysql-connections.rb as the basis for mine, and its header is clearly wrong. Thanks!

**Pull request closed without merging**